docs/esa/euclid/euclid.rst

```diff
@@ -1187,45 +1187,48 @@ The query below retrieves a random sample of Euclid sources having spectra.
+                  161                        6170 /data/euclid_q1/Q1_R1/SIR/102159190  66.2258188827905        4 EUC_SIR_W-COMBSPEC_102159190_2024-11-05T16:27:45.906227Z.fits ...      1672 267.868196199085 2678681961662258188              66254               1
+                  161                        6170 /data/euclid_q1/Q1_R1/SIR/102159190  66.2302887578947        3 EUC_SIR_W-COMBSPEC_102159190_2024-11-05T16:27:45.906227Z.fits ...      2121 268.121362468449 2681213624662302887              66283               1
+                  161                        6168 /data/euclid_q1/Q1_R1/SIR/102159190  66.2180692869346        3 EUC_SIR_W-COMBSPEC_102159190_2024-11-05T16:27:08.615108Z.fits ...       320 267.232306764858 2672323067662180692              65811               1
+                  161                        6168 /data/euclid_q1/Q1_R1/SIR/102159190  66.2163449302499        4 EUC_SIR_W-COMBSPEC_102159190_2024-11-05T16:27:08.615108Z.fits ...       678 267.390646814535 2673906468662163449              65836               1
+                  161                        6168 /data/euclid_q1/Q1_R1/SIR/102159190  66.2177524156252        4 EUC_SIR_W-COMBSPEC_102159190_2024-11-05T16:27:08.615108Z.fits ...       787 267.444077136233 2674440771662177524              65843               1
+                  ...                         ...                                 ...               ...      ...                                                           ... ...       ...              ...                 ...                ...             ...
+                  180                        7517 /data/euclid_q1/Q1_R1/SIR/102042287 -29.5639947358353        2 EUC_SIR_W-COMBSPEC_102042287_2024-11-05T17:42:15.562399Z.fits ...        58 54.5720898250079 -545720898295639947             264701               1
+                  161                        6203 /data/euclid_q1/Q1_R1/SIR/102159190  66.2431257852549        2 EUC_SIR_W-COMBSPEC_102159190_2024-11-05T16:34:19.265698Z.fits ...        88 267.227375121186 2672273751662431257              67437               1
   Length = 2000 rows
+  >>> print("source ids:")
+  >>> print(results['source_id'])
+  <Column name='source_id' dtype='int64' length=200>
+  2672611464662272115
+  2673193314662304322
+  2673997437662259968
+  2676108358662258832
+                  ...
+  2671568248661962869
+  2673081407661969815
+  2673195636661993590
 
 
 The following example shows how to retrieve the DataLink products (1D Spectra) associated with the previous sources (IDs).
 
 .. Skipping authentication requiring examples
 .. doctest-skip::
 
-  >>> files = Euclid.get_spectrum(source_id='2675005060662306333')
+  >>> files = Euclid.get_spectrum(retrieval_type='SPECTRA_BGS', source_id='2675005060662306333')
   >>> from astropy.io import fits
   >>> print(fits.info(files[0]))
```

Comment:
The actual value will never pass the test as the location will be different, so we can add the ignore in here now.
```suggestion
  >>> print(fits.info(files[0]))  # doctest: +IGNORE_OUTPUT
```